src/behaviors/Behavior.js

```diff
@@ -6,7 +6,7 @@ var Base = require('../lib/Base');
 
 var Column = require('./Column');
 var dialogs = require('../dialogs');
-var CellProvider = require('../lib/CellProvider');
+var CellProvider = require('../cellRenderers/CellProvider');
```

Comment:
If you intention in moving this to a new `cellRenderers` folder was to break out the various cell renderer functions into separate files, please go ahead and do so.